metadata/schema_test.go

```diff
@@ -249,17 +275,17 @@ func TestGetSchema(t *testing.T) {
 	assert.Nil(t, schema, "should return no schema for bad type")
 
 	schema, err = GetSchema(badArrayType, components)
-	_, expectedErr = buildArraySchema(reflect.New(badArrayType).Elem(), components)
+	_, expectedErr = buildArraySchema(reflect.New(badArrayType).Elem(), components, false)
 	assert.EqualError(t, err, expectedErr.Error(), "should return error when build array errors")
 	assert.Nil(t, schema, "should return no schema when build array errors")
 
 	schema, err = GetSchema(badSliceType, components)
-	_, expectedErr = buildSliceSchema(reflect.MakeSlice(badSliceType, 1, 1), components)
+	_, expectedErr = buildSliceSchema(reflect.MakeSlice(badSliceType, 1, 1), components, false)
 	assert.EqualError(t, err, expectedErr.Error(), "should return error when build slice errors")
 	assert.Nil(t, schema, "should return no schema when build slice errors")
 
 	schema, err = GetSchema(badMapItemType, components)
-	_, expectedErr = buildMapSchema(reflect.MakeMap(badMapItemType), components)
+	_, expectedErr = buildMapSchema(reflect.MakeMap(badMapItemType), components, false)
```

Comment:
should I be able to see a buildMapSchema(..., true) thing somewhere in this test file?

Reply:
These are just tests that the errors occur the TestGetSchema has all the different cases of handling the nested true vs false